guides/common/modules/proc_configuring-logging-to-journal-or-file-based-logging.adoc

```diff
@@ -0,0 +1,42 @@
+[id="Configuring_Logging_to_Journal_or_File-based_Logging_{context}"]
+= Configuring Logging to Journal or File-based Logging
+
+{Project} uses file-based logging by default.
+You can use the `{foreman-installer}` command to reconfigure logging.
+
+.Procedure for configuring logging with Journal
+* Enter the following `{foreman-installer}` command to configure logging to the `journald` service:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} \
+--foreman-logging-layout pattern \
```

Comment:
Not for you to fix, but this makes me wonder if we can improve the installer to not need this. https://projects.theforeman.org/issues/36582

Reply:
Ok, resolving this conversation as there is no action item for the docs update.